Earthfile

```diff
@@ -475,7 +475,8 @@ earthly-docker:
 # if no dockerhub mirror is not set it will attempt to login to dockerhub using the provided docker hub username and token.
 # Otherwise, it will attempt to login to the docker hub mirror using the provided username and password
 earthly-integration-test-base:
-    FROM +earthly-docker
+    ARG BUILDKIT_PROJECT
```

Comment:
nit: I'm not even sure the arg definition is necessary because I think --pass-args would just pass the value from the caller but maybe it's good to be explicit

Reply:
That's true. I removed it.